### PR TITLE
Explain options in "Getting Started"

### PR DESCRIPTION
**Pull request by brentvatne**
As discussed in Discord, this is a WIP pull request to improve the wording in "Getting Started" to help people understand the differences between using Expo CLI and React Native CLI to get a "Hello World" app running.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -39,7 +42,7 @@ title: Getting Started
       display: block;
       border-bottom-color: #05A5D1;
       border-radius: 3px;
-      margin: 2px 0px 2px 0px;
+      margin: 2px 0px 2px 0px !important;
```
Comment (brentvatne) [inline docs/getting-started.md]: this fixes toggle alignment on mobile, it seems that `toggler` is also a class that is defined in docusaurus and that style overrides this one

**Comment by hramos** [comment]
These changes look great to me. Thanks for doing this!

**Comment by brentvatne** [comment]
@cpojer done

**Comment by charpeni** [comment]
Oops, tests were failing because versioned getting started has been renamed.

I'll fix this!

**Comment by charpeni** [comment]
Fixed in 8d54fb0d69d409f5bc56e64d86851618984d80b5, should be deployed in a few minutes.

Good work @brentvatne! 🎉

**Pull request merged**